src/components/Dropdown/Dropdown.tsx

```diff
@@ -294,13 +313,16 @@ export interface DropdownProps {
   variant?: variants;
   optionsVariant?: variants;
   valueVariant?: variants;
+  valueCountVariant?: variants;
 
   shouldStayInView?: boolean;
   intersectionThreshold?: number;
   intersectionContainer?: HTMLElement | null;
   intersectionObserverPrecision?: number;
   virtualizeOptions?: boolean;
 
+  showCloseIcon?: boolean;
```

Comment:
this should be covered by the clearable prop - there shouldn't be a use-case where the dropdown is clearable but the clear button isn't shown (making it not clearable)

Reply:
Good call I agree. Changing now

Reply:
It looks like clearable is appended onto DropdownProps in the story. Removing from there and adding to DropdownProps.

Reply:
cool